prepare-assets.py

```
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+# Prepare assets for deployment; remove unnecessary assets to decrease bundle size
+
+import os, sys
+
+if "--default" in sys.argv:
+    os.system("rm -rf ./public/pro")
```

Ruff (0.12.2)

Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)